docs/.vuepress/handsontable-manager/dependencies.js

```diff
@@ -0,0 +1,101 @@
+// eslint-disable-next-line no-restricted-globals
+const isBrowser = (typeof window !== 'undefined');
+
+const formatVersion = version => (/^\d+\.\d+$/.test(version) ? version : 'latest');
+const getHotUrls = (version) => {
+  if (version === 'next' && isBrowser) {
+    return {
+      handsontableJs: '/docs/handsontable.js',
+      handsontableCss: '/docs/handsontable.css',
+      languagesJs: 'https://cdn.jsdelivr.net/npm/handsontable/dist/languages/all.js'
+    };
+  }
+
+  const mappedVersion = formatVersion(version);
+
+  return {
+    handsontableJs: `https://cdn.jsdelivr.net/npm/handsontable@${mappedVersion}/dist/handsontable.full.min.js`,
+    handsontableCss: `https://cdn.jsdelivr.net/npm/handsontable@${mappedVersion}/dist/handsontable.full.min.css`,
+    languagesJs: `https://cdn.jsdelivr.net/npm/handsontable@${mappedVersion}/dist/languages/all.js`
+  };
+};
+const getFixer = () => {
+  if (isBrowser) {
+    // eslint-disable-next-line no-restricted-globals
+    return [`${window.location.origin}/docs/scripts/fixer.js`, ['require', 'exports']];
+  }
+
+  return ['https://handsontable.com/docs/scripts/fixer.js', ['require', 'exports']];
+};
+
+/**
+ * Some further version of Handsontable will needs different version of dependencies.
+ * The function `buildDependencyGetter` is the best place to care about that.
+ *
+ * @param {string} version The current selected documentation version.
+ * @returns {Function} Returns a function factory with the signature
+ *                     `{function(dependency: string): [string,string[],string]} [jsUrl, dependentVars[]?, cssUrl?]`.
+ */
+const buildDependencyGetter = (version) => {
+  const { handsontableJs, handsontableCss, languagesJs } = getHotUrls(version);
+  const mappedVersion = formatVersion(version);
+  const fixer = getFixer();
+
+  return (dependency) => {
+    /* eslint-disable max-len */
+    const dependencies = {
+      fixer,
+      hot: [handsontableJs, ['Handsontable'], handsontableCss],
+      react: ['https://unpkg.com/react@17/umd/react.development.js', ['React']],
+      'react-dom': ['https://unpkg.com/react-dom@17/umd/react-dom.development.js', ['ReactDOM']],
+      'hot-react': [`https://cdn.jsdelivr.net/npm/@handsontable/react@${mappedVersion}/dist/react-handsontable.js`, ['Handsontable.react']],
+      'react-redux': ['https://cdnjs.cloudflare.com/ajax/libs/react-redux/7.2.4/react-redux.min.js'],
+      numbro: ['https://handsontable.com/docs/8.3.2/components/numbro/dist/languages.min.js', ['numbro.allLanguages', 'numbro']],
+      redux: ['https://cdn.jsdelivr.net/npm/redux@4/dist/redux.min.js', []],
+      rxjs: ['https://cdn.jsdelivr.net/npm/rxjs@6/bundles/rxjs.umd.js', [/* todo */]],
+      'core-js': ['https://cdn.jsdelivr.net/npm/core-js@2/client/core.min.js', [/* todo */]],
+      zone: ['https://cdn.jsdelivr.net/npm/zone.js@0.9/dist/zone.min.js', [/* todo */]],
```

Comment:
This array contains variables name to remove when the dependencies are unloaded - it is potentially a memory leak here.

On the other side, we do not observe any problems with this. IMO it might be done as a separate issue in the next sprint.

Reply:
Ok